File: week-02/day-01/seconds_in_a_day.py
current_hours = 14;
current_minutes = 34;
current_seconds = 42;

# Write a program that prints the remaining seconds (as an integer) from a
# day if the current time is represented bt the variables
day_hours = 24
day_minutes = 00
day_seconds = 00

remaining_hours = day_hours - current_hours
remaining_minutes = day_minutes - current_minutes
remaining_seconds = day_seconds - current_seconds
remaining = remaining_hours*60*60 + remaining_minutes*60
remaining_total = remaining + remaining_seconds
remaining_total_var = int(remaining_total)
print( remaining_total_var ) 

File: week-07/day-01/seconds_in_a_day.js
'use strict';

let currentHours = 14;
let currentMinutes = 34;
let currentSeconds = 42;

// Write a program that prints the remaining seconds (as an integer) from a
// day if the current time is represented by these variables

let dayHours = 24;
let dayMinutes = 0;
let daySeconds = 0;
let remainingHours = dayHours - currentHours;
let remainingMinutes = dayMinutes - currentMinutes;
let remainingSeconds = daySeconds - currentSeconds;
let remaining = remainingHours * 60 * 60 + remainingMinutes * 60
let remainingTotal = remaining + remainingSeconds

console.log(remainingTotal);